README.rst

```diff
@@ -109,3 +109,21 @@ There is a simple ``docker-compose.yml`` file to make it easier to setup a
 database for development.
 
 You can also setup the database on your own.
+
+Database Migration
+^^^^^^^^^^^^^^^^^^
+
+For database migration we are using `Alembic <https://alembic.sqlalchemy.org/en/latest/>_.
+If you have setup a new database on the latest database schema, just execute
+
+.. code-block:: bash
+
+   alembic stamp head
+
+to add the alembic metadata to the database and make it ready for later migration.
+
+If you want to migrate your database to the latest version, just execute
+
+.. code-block:: bash
+
+   alembic upgrade head
```

Comment:
Maybe add something like:

"Note: you need to have emissions-api installed or you need to change the import path of `emissionsapi.db` in `alembic/env.py`."

But that was just a problem I was running in to.